kaspad/src/args.rs

```diff
@@ -74,7 +76,7 @@ pub struct Args {
 impl Default for Args {
     fn default() -> Self {
         Self {
-            appdir: Some("datadir".into()),
+            appdir: None,
```

Comment:
Why was this change necessary?

Reply:
a residual from testing the defaults issue, reverted.